docs/flow/unidist/config.rst

```diff
@@ -0,0 +1,68 @@
+..
+      Copyright (C) 2021-2022 Modin authors
+
+      SPDX-License-Identifier: Apache-2.0
+
+:orphan:
+
+Unidist Configuration Settings
+""""""""""""""""""""""""""""""
+
+Using these configuration settings, the user can tune unidist's behavior. Below you can find
+configuration settings currently provided by unidist.
+
+Public API
+''''''''''
+
+Potentially, the source of configuration settings can be any, but for now only environment
```

Comment:
What do you mean by "the source of configuration settings can be any"?

Reply:
a file, for instance